apps/labs/pages/api/annual-report.pdf.ts

```diff
@@ -0,0 +1,37 @@
+import type { NextRequest } from 'next/server';
+
+export const config = {
+  runtime: 'experimental-edge',
+};
+
+export default async function handler(req: NextRequest) {
+  // TODO - remove after checking the logs on Vercel
+  console.log(`
+  req.url ${req.url}
+  req.referer ${req.headers.get('referer')}
+  req.ip ${req.ip}
+  user-agent ${req.headers.get('user-agent')}
+  `);
+  // The shape of this fetch comes from the Plausible docs:
+  // https://plausible.io/docs/events-api
+  await fetch('https://plausible.io/api/event', {
+    method: 'POST',
+    headers: {
+      'User-Agent': req.headers.get('user-agent'),
+      'X-Forwarded-For': req.ip,
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      domain: 'gabalafou.com',
```

Comment:
Is this meant to be a placeholder domain? I assume this will need to be swapped to the labs domain @gabalafou

Reply:
Good catch! Yes. This was me trying not to pollute our labs.quansight.org, but eventually I gave up because Plausible doesn't really support a dev-only account. You gotta pay. 
```suggestion
      domain: 'labs.quansight.org',
```